sklearn/linear_model/coordinate_descent.py

```diff
@@ -1063,6 +1063,8 @@ def fit(self, X, y):
 
         if hasattr(self, 'l1_ratio'):
             model_str = 'ElasticNet'
+            if self.l1_ratio == 0:
+                raise ValueError("l1_ratio = 0 not supported")
```

Comment:
Sorry, that was sloppy of me.  If 0 was contained in the list, the code would have run without an error, but the `mse_path_` on the estimator for that value, would all be `nan`s. I.e. a user would think the ridge estimate was evaluated, when it was in fact not. I updated the error message.